packages/http/src/mocker/negotiator/NegotiatorHelpers.ts

```diff
@@ -125,11 +125,17 @@ const helpers = {
         },
         httpContent
       );
+    } else {
+      // no httpContent found, returning empty body
+      return {
+        code,
+        mediaType: 'text/plain',
+        example: {
+          externalValue: '',
```

Comment:
I think this should not be an `INodeExternalExample`, but a `INodeExample`, since this is not coming from an external source but rather generated here internally.

Moreover, I think it'd be more correct if the value would be `undefined`

```suggestion
          value: undefined,
```

Reply:
Good spot, INodeExample was sth I've been looking for but for unknown reason did not found it. Also returning undefined.